### PR TITLE
Add incident report for 02-03-2022

### PR DESCRIPTION
**Pull request by tony-griffin**
Draft for incident report 02-03-2022

### CODE REVIEW

File: sidebars.js [hunk 1]
```diff
@@ -47,5 +47,22 @@ module.exports = {
...
       ],
     },
+    {
+      type: "category",
+      label: "Incidents",
+      items: [
+        {
+          type: "category",
+      label: "2022",
+      items: [
```
Comment (acodeninja) [inline sidebars.js:57]: Maybe just a quick tidy up here.

File: docs/incidents/2022/02-03-2022.md [hunk 2]
```diff
@@ -0,0 +1,274 @@
+# Incident Report 02-03-2022
```
Comment (acodeninja) [inline docs/incidents/2022/02-03-2022.md:1]: You'll maybe need some front matter in here?

**Review by acodeninja — CHANGES_REQUESTED** [review]
Two small changes, but otherwise looking good 👍

File: sidebars.js (hunk 1 above)
Comment (tony-griffin) [reply sidebars.js:57]: Good spot!

File: docs/incidents/2022/02-03-2022.md (hunk 2 above)
Comment (tony-griffin) [reply docs/incidents/2022/02-03-2022.md:1]: Added, thanks.

**Review by acodeninja — APPROVED** [review]
Awesome, thank you.

**Pull request merged**